src/app/Form/FormKeyInfos/FormKeyInfos.js

```diff
@@ -1,175 +1,197 @@
+            value={formInputs.shots}
+            onChange={(event) =>
+              handleChange(event.target.name, event.target.value)
+            }
+            onBlur={() => updateDirtyInputs('shots')}
+          />
+        </label>
+        <Note>{showErrorMessage('shots')}</Note>
+      </KeyInfos>
+      <ButtonWrapper>
+        <Button
+          type="button"
+          onClick={handleCancel}
+          iconComponent={<CancelIconPrimaryText />}
+          text="Cancel"
+          data-testid="button-cancel"
+        />
+        <Button
+          main
+          disabled={!isSaveButtonShown}
+          iconComponent={<CheckIconLightText />}
+          text="Save"
+          data-testid="button-save"
+        />
+      </ButtonWrapper>
+      <Note>
+        *Please do not clear your browsers cache, in order to permanently save
+        your game details
+      </Note>
+    </Form>
+  )
 }
 
 const Form = styled.form`
-    margin: 10px 20px;
-    padding: 10px;
-    display: grid;
-    place-items: center;
-    border-radius: 25px;
-    box-shadow: 0 0 10px var(--primary-medium);
-    background-color: var(--text-light);
-    color: var(--text-dark);
-    font-size: 1rem;
+  margin: 10px 20px;
+  padding: 10px;
+  display: grid;
+  place-items: center;
+  border-radius: 25px;
+  background-color: var(--light);
+  font-size: 1rem;
+
+  input::-webkit-outer-spin-button,
+  input::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
 
-    input::-webkit-outer-spin-button,
-    input::-webkit-inner-spin-button {
-        -webkit-appearance: none;
-        margin: 0;
-    }
-    
-    input[type=number] {
-        -moz-appearance: textfield;
-    }
+  input[type='number'] {
+    -moz-appearance: textfield;
+  }
 
-    Button {
-        margin-top: 15px;
-        margin-bottom: 5px;
-    }
-`
-const Headline = styled.h3`
-    text-align: center;
-    text-transform: uppercase;
+  Button {
+    margin-top: 15px;
+    margin-bottom: 5px;
+  }
 `
 const KeyInfos = styled.fieldset`
-    margin: 0;
-    padding: 0;
-    border: none;
-    display: grid;
-    width: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: grid;
+  border: none;
 
-    label {
-        margin-top: 10px;
-        margin-bottom: 5px;
-        font-family: 'Raleway', sans-serif;
-    }
+  label {
+    margin-top: 10px;
+    margin-bottom: 5px;
+  }
 
-    input {
-        display: block;
-        margin-top: 3px;
-        padding: 5px;
-        width: 100%;
-        border-style: none;
-        border-bottom: 1px solid var(--primary-dark);
-        color: var(--primary-dark);
-        font-family: 'Montserrat', sans-serif;
-    }
+  input {
+    width: 100%;
+    margin-top: 3px;
+    padding: 5px;
+    display: block;
+    border-style: none;
+    border-bottom: 1px solid var(--secondary-medium);
+    color: var(--secondary-medium);
+    font-family: 'Montserrat', sans-serif;
+  }
+
+  input::placeholder {
+    font-style: italic;
```

Comment:
Isn't there a defined font style for` italic`? The font could possibly be displayed incorrectly.